framework/console/controllers/MigrateController.php

```diff
@@ -363,10 +380,15 @@ protected function parseFields()
 
             foreach ($chunks as $i => &$chunk) {
                 if (strpos($chunk, 'foreignKey') === 0) {
-                    preg_match('/foreignKey\((\w*)\)/', $chunk, $matches);
-                    $foreignKeys[$property] = isset($matches[1])
-                        ? $matches[1]
-                        : preg_replace('/_id$/', '', $property);
+                    preg_match('/foreignKey\((\w*)\s?(\w*)\)/', $chunk, $matches);
```

Comment:
Regular expression changed for support setting field name for reference key by using whitespace like this: ```foreignKey(table_name reference_key)```

Now FK may by defined in three ways:
```table_id:foreignKey(table_name reference_key)```
```table_id:foreignKey(table_name)```
```table_id:foreignKey```